### ISSUE TITLE
'.' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by maheenur13** [nuejs/create-nue#8]
when i try to start the project give me this error!

```
$ npm run start

> create-nue@0.1.3 start
> npm run minify && npm run render && npm run compile && npm run serve


> create-nue@0.1.3 minify
> ./scripts/minify.js

'.' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by luminarious** [comment]
To make it work on Windows I explicitly set them to run with node

````
    "start": "npm run minify && npm run render && npm run compile && npm run serve",
    "serve": "cd www && node ../scripts/server.js",
    "compile": "node ./scripts/compile.js",
    "minify": "node ./scripts/minify.js",
    "render": "node ./scripts/render.js"
```

**Comment by diomed** [comment]
> To make it work on Windows I explicitly set them to run with node
> 
> ```
> "start": "npm run minify && npm run render && npm run compile && npm run serve",
> "serve": "cd www && node ../scripts/server.js",
> "compile": "node ./scripts/compile.js",
> "minify": "node ./scripts/minify.js",
> "render": "node ./scripts/render.js"
> ```

where to change this, bro?

**Comment by diomed** [comment]
@tipiirai is this not an issue worth of attention?

windows users casually being ignored

**Comment by kon-pas** [comment]
Copy and paste the code above to `scripts` in `package.json`. It is a very quick fix until we figure out the solution to satisfy all users.

Here is the whole `package.json`, if you are still having troubles:
```json
{
  "name": "create-nue",
  "type": "module",
  "version": "0.1.3",
  "homepage": "https://github.com/nuejs/create-nue",
  "description": "A recommended way to start a Nue project",
  "repository": {
    "url": "https://github.com/nuejs/create-nue.git",
    "type": "git"
  },
  "bin": {
    "create-nue": "./scripts/init.js"
  },
  "scripts": {
    "start": "npm run minify && npm run render && npm run compile && npm run serve",
    "serve": "cd www && node ../scripts/server.js",
    "compile": "node ./scripts/compile.js",
    "minify": "node ./scripts/minify.js",
    "render": "node ./scripts/render.js"
  },
  "dependencies": {
    "nuejs-core": "^0.1.0",
    "js-yaml": "^4.1.0"
  }
}
```

**Comment by diomed** [comment]
thank you @kon-pas ; that helped

**Comment by maheenur13** [comment]
thanks @kon-pas

**Comment by tipiirai** [comment]
Fixed with #16 

Thanks guys!
